OPERATOR.rst

```diff
@@ -640,14 +640,15 @@ downloads the tarball from the package registry and unpacks that tarball to the
 S3 bucket backing the Data Browser's CloudFront distribution.
 
 Typically, CC requests the deployment of a Data Browser instance on Slack,
-specifying the commit they wish to be deployed. After the system administrator
-approves that request, the operator merges the specified commit into one of the
-``ucsc/{atlas}/{deployment}`` branches and then pushes that branch to the
-``DataBiosphere/data-browser`` project on GitHub, and the ``ucsc/data-browser``
-project on the GitLab instance for the Azul ``{deployment}`` that backs the Data
-Browser instance to be deployed. For the merge commit title, SmartGit's default
-can be used, as long as the title reflects the commit (branch, tag, or sha1)
-specified by CC.
+specifying the commit (branch, tag, or sha1) they wish to be deployed. After the
+system administrator approves that request, the operator first pushes the
+specified tag to the appropriate GitLab instance(s) (if a tag was specified),
+then merges the specified commit into one of the ``ucsc/{atlas}/{deployment}``
+branches. Then, that branch is pushed to the ``DataBiosphere/data-browser``
+project on GitHub, and the ``ucsc/data-browser`` project on the GitLab instance
+for the Azul ``{deployment}`` that backs the Data Browser instance to be
```

Comment:
This is very specific (I think I wrote it) whereas the part about pushing the tag is wishy-washy ("appropriate"). This is very confusing since it is not clear that both statements refer to the same GitLab instance. Please think about a better way to word this and request to discuss your proposal in PL.